aspnetcore/blazor/hosting-models.md

```diff
@@ -88,8 +88,6 @@ Blazor WebAssembly includes support for trimming unused code from .NET Core fram
 
 :::moniker-end
 
-:::moniker range=">= aspnetcore-6.0"
```

Comment:
It was just due to our expansion of the table in the next section to include Blazor Hybrid in a topic that isn't fully versioned (i.e., this topic doesn't use Whole-topic Versioning). I'll version the whole topic so that Blazor Hybrid will only appear for 6.0 or later. I'll do it first thing Wednesday morning.

Reply:
Ah! I see why ... your edit to the original PR ...

https://github.com/dotnet/AspNetCore.Docs/pull/24965/commits/4ddaf34dd78d126d805d0c1ebce9287aa2601c0f

Line 133.